go.mod

```diff
@@ -18,7 +18,8 @@ require (
 	github.com/iancoleman/strcase v0.2.0
 	github.com/martinlindhe/base36 v1.1.1
 	github.com/open-policy-agent/opa v0.52.0
-	github.com/pomerium/pomerium v0.20.1-0.20230502181647-150c241f4948
+	github.com/pomerium/csrf v1.7.0
```

Comment:
I'm not sure I understand why this changed from an indirect to a direct dependency without any changes to the Go code.